README.md

```diff
@@ -276,6 +276,33 @@ Suggested attribute mappings:
 |`Email`|`user.email`|
 
 
+**Auth Proc Filter Hooks**
+
+Other plugins may hook into SAML2 and create custom Auth Proc Filters.
+Auth Proc Filters allows you to do stuff at the IdP after authentication is complete
```

Comment:
This isn't quite correct

"Auth Proc Filters allows you to do stuff at the IdP after authentication is complete"

something like

"Auth Proc Filters allows you to mutate of the attributes passed back from the IdP before Moodle handles them and does the profile field mappings"